custom_components/aux_cloud/api/const.py

```diff
@@ -23,11 +23,11 @@ class AuxProductCategory(auto):
 
 # Common constants
 AUX_MODE = "ac_mode"
-AUX_MODE_AUTO = {AUX_MODE: 0}
-AUX_MODE_COOLING = {AUX_MODE: 1}
+AUX_MODE_COOLING = {AUX_MODE: 0}
+AUX_MODE_HEATING = {AUX_MODE: 1}
 AUX_MODE_DRY = {AUX_MODE: 2}
 AUX_MODE_FAN = {AUX_MODE: 3}
-AUX_MODE_HEATING = {AUX_MODE: 4}
+AUX_MODE_AUTO = {AUX_MODE: 4}
```

Comment:
I tested it and you're right, those are correct for ACs. My bad. For heat pumps however heat=4 as I have it set right now.

Could you please update it to resemble:

```suggestion
AC_MODE_COOLING = {AUX_MODE: 0}
AC_MODE_HEATING = {AUX_MODE: 1}
AC_MODE_DRY = {AUX_MODE: 2}
AC_MODE_FAN = {AUX_MODE: 3}
AC_MODE_AUTO = {AUX_MODE: 4}
```

in Heat pump section add
```python
HP_MODE_AUTO = {AUX_MODE: 0}
HP_MODE_COOLING = {AUX_MODE: 1}
HP_MODE_HEATING = {AUX_MODE: 4}
```

and update where applicable?

The `AUX_MODE = "ac_mode"` parameter name is the same for both heat pump and ACs

Reply:
Actually, I'll merge it and rework the param names a little. Will try to fix the issue where entities are not provided when home assistant restarts/integration reloads with some of the devices being offline